js/app/test/audio_component_events.spec.ts

```diff
@@ -9,7 +9,7 @@ test("Audio click-to-upload uploads audio successfuly. File downloading works an
 	const uploader = await page.locator("input[type=file]");
 	await Promise.all([
 		uploader.setInputFiles(["../../test/test_files/audio_sample.wav"]),
-		page.waitForResponse("**/upload?*")
+		page.waitForResponse("**/upload*?*")
```

Comment:
Some had "**/upload?*" and some had "**/upload*?*". Using the latter just for consistency.